Makefile

```diff
@@ -5,6 +5,8 @@ all: build
 generate_dto:
 	@./mvnw compile
 	@npm install --prefix ../E_User-Tech
+	@npm install --prefix ../E_User-Tech
+	@rm -rf ../E_Admin/src/app/generated && mkdir ../E_Admin/src/app/generated && cp ../E_User-Tech/src/app/generated/dto.ts ../E_Admin/src/app/generated/dto.ts
```

Comment:
make this into two separate commands or else mkdir will not be executed when .../generated does not exist